tests/test_reflection_views.py

```diff
@@ -40,3 +40,25 @@ def test_late_binding_view_reflection(redshift_engine):
     view = Table('my_late_view', MetaData(),
                  autoload=True, autoload_with=redshift_engine)
     assert(len(view.columns) == 2)
+
+
+def test_late_binding_view_reflection(redshift_engine):
```

Comment:
@briandailey can you change this name to `test_spectrum_reflection` or something similar? name clashing in tests.